blueprints/-addon-import.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
Right. This was something I was a bit curious about while I was doing the work, because it seems like I should only want to create native class components and post qunit-rfc test files (plus the general blueprints for models, etc., which wouldn't change much). 

In the other comments you've made, I follow and agree. I wasn't sure where the registry was and wasn't necessary, so I'll go back and check for that. but there's one thing I'm not sure about, and I'll comment on that separately. 

Thanks!